<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Beta</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="mainTabs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="ownerTab">
     <attribute name="title">
      <string>Proprietários</string>
     </attribute>
     <widget class="QTableWidget" name="ownerWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>1041</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1041</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Proprietários</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="createOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>470</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Novo Proprietário</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>590</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover Proprietário</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>530</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Alterar Dados de Prop.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="readOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>250</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar por Nome</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchNameLine">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>220</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshOwnerBtn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>10</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>80</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>440</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>430</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="birdTab">
     <attribute name="title">
      <string>Pássaros</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1241</width>
        <height>641</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Bicudo Fibra</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bicudo Fibra</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="bicudoWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshBicudoBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createBicudoBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Bicudo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteBicudoBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Bicudo</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateBicudoBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Alterar Dados de Bicudo</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Curió Fibra</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Curió Fibra</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="updateCuriofBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Alterar Dados de Curió</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteCuriofBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Curió</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createCuriofBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Curió</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="curiofWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshCuriofBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Chanchão Fibra</string>
       </attribute>
       <widget class="QPushButton" name="updateChanchaoBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Alterar Dados de Chanchão</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshChanchaoBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createChanchaoBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Chanchão</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Chanchão Fibra</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="chanchaoWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteChanchaoBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Chanchão</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Coleiro</string>
       </attribute>
       <widget class="QTableWidget" name="coleiroWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Coleiro</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="updateColeiroBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Alterar Dados de Coleiro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshColeiroBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteColeiroBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Coleiro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createColeiroBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Coleiro</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Trinca Ferro</string>
       </attribute>
       <widget class="QPushButton" name="deleteTrincaBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Trinca Ferro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="createTrincaBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Trinca Ferro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateTrincaBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Alterar Dados de T. Ferro</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshTrincaBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Trinca Ferro</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="trincaWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Curió Canto-Livre</string>
       </attribute>
       <widget class="QPushButton" name="updateCuriolBtn">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Alterar Dados de Curió</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="curiolWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>1221</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="createCuriolBtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Novo Curió</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Curió Canto-Livre</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteCuriolBtn">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remover Curió</string>
        </property>
       </widget>
       <widget class="QPushButton" name="refreshCuriolBtn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>550</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Atualizar</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="chanchaoTab">
     <attribute name="title">
      <string>Competição</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>161</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Configurações</string>
       </attribute>
       <widget class="QPushButton" name="createCompBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>411</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Nova Competição</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reloadCompBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Recarregar Competição</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboReloadComp">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>50</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboDelComp">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>90</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteCompBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Excluir Competição</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>1241</width>
        <height>471</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Relações</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>421</width>
        <height>191</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Inscrições</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>12</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Raça:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboRace">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>6</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Bicudo Fibra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Curió Fibra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chanchão Fibra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Coleiro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trinca Ferro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Curió Canto-Livre</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboOwner">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>44</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Proprietário:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBird">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>82</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>88</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Pássaro:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="subscribeCompBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Increver</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
